Monika After Story/game/screens.rpy

```diff
@@ -2597,86 +2597,164 @@ define main_adj = ui.adjustment()
+                                action Return(i_label)
 
-                null height 20
+                if cat_length != 1:
+                    null height 20
+
+                    if cat_length == 0:
+                        textbutton _("Nevermind.") action [Return(False), Function(store.prev_adj.change, 0)]
+
+                    elif cat_length > 1:
+                        textbutton _("Go Back") action [Return(-1), Function(store.prev_adj.change, 0)]
+
+        # Right panel
+        if main_items:
+            fixed:
+                area right_area
+
+                bar:
+                    adjustment main_adj
+                    style "classroom_vscrollbar"
+                    xalign right_align
 
-                textbutton _("Nevermind.") action [Return(False), Function(store.prev_adj.change, 0)]
+                vbox:
+                    ypos 0
+                    yanchor 0
+
+                    viewport:
+                        yadjustment main_adj
+                        yfill False
+                        mousewheel True
+                        arrowkeys True
+
+                        vbox:
+                            for i_caption, i_label in main_items:
+                                textbutton i_caption:
+                                    if renpy.has_label(i_label) and not seen_event(i_label):
+                                        style "twopane_scrollable_menu_new_button"
+
+                                    elif not renpy.has_label(i_label):
+                                        style "twopane_scrollable_menu_special_button"
+
+                                    action [Return(i_label), Function(store.prev_adj.change, 0)]
+
+                    null height 20
+
+                    textbutton _("Nevermind.") action [Return(False), Function(store.prev_adj.change, 0)]
+
+    # Search bar
+    # The constants are hardcoded, but the menu looks good so just don't change them
+    frame:
+        xpos left_area[0]
+        ypos left_area[1] - 55
+        xsize right_area[0] - left_area[0] + right_area[2]# 530
+        ysize 40
+        background Solid("#ffaa99aa")
+
+        viewport:
+            draggable False
+            arrowkeys False
+            mousewheel "horizontal"
+            xsize right_area[0] - left_area[0] + right_area[2] - 10
+            ysize 38
+            xadjustment ui.adjustment(ranged=store.mas_ui.twopane_menu_adj_ranged_callback)
+
+            input:
+                id "search_input"
+                style_prefix "input"
+                length 50
+                xalign 0.0
+                layout "nobreak"
+                # allow "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789 _#"
+                changed store.mas_ui.twopane_menu_search_callback
+
+        if flt_evs is None:
+            text "Search for a conversation...":
+                text_align 0.0
+                layout "nobreak"
+                color "#FFFFFF"
```

Comment:
Tried some style changes and was satisfied with this. This makes the ghost text look more like real ghost text.
```suggestion
                color "#EEEEEEB2"
                first_indent 10
                line_leading 1
                outlines []
```